### DISCUSSION TITLE
blog/reflection-on-emergency-broadcasting

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#34] [Announcements]
# blog/reflection-on-emergency-broadcasting

哲学片段与学术之路

https://pathos.page/blog/reflection-on-emergency-broadcasting

**Comment by tjsky** [comment]
一个社会学常态：“下级一定会用最有利于自己的方式，解读并执行上级的政策”。
当地发现利用起来搞利己行为后没引起反弹了，既然发现是低风险高收益的政绩利益渠道，不去占这个“便宜”的人是很少的。
现实中，越是本地土著刺头多动不动就投诉大闹的地方，政府越是服务态度良好。

**Comment by yyqEternal** [comment]
疫情的时候，我老家安徽皖北穷贫困县某农村，有人靠关系拿到了通行证，我们上街买东西要看村里证明，有人拿了几十张分给别人，被某个人直接在喇叭全庄光播指名道姓的骂，其他的时候也就播播广播核酸确实烦人后来别人也不想搞封太久了直接不播了，农忙的时候播不让焚烧秸秆

**Reply by Pathsis** [reply]
现在不播了吗？我老家也是皖北。现在我这个市真是恶心死了。以后全省都学我这边，真是灾难。

**Reply by yyqEternal** [reply]
我是阜阳某贫困县的，你现在在哪个市了？我老家这边早就不广播了，应该说从22年3月开始彻底不广播核酸也是靠村里人互相通知
